src/VS.Web.CG.Mvc/Templates/Identity/Services/EmailSender.cshtml

```diff
@@ -0,0 +1,18 @@
+@inherits Microsoft.VisualStudio.Web.CodeGeneration.Templating.RazorTemplateBase
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace @(Model.Namespace).Areas.Identity.Services
+{
+    // This class is used by the application to send email for account confirmation and password reset.
+    // For more details see https://go.microsoft.com/fwlink/?LinkID=532713
+    public class EmailSender : IEmailSender
+    {
+        public Task SendEmailAsync(string email, string subject, string htmlMessage)
+        {
+            return Task.CompletedTask;
```

Comment:
This method appears to be missing its functionality. Should this maybe be an abstract class or method?

Reply:
@javiercn / @danroth27 any thoughts?